src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/RMSMappingQuality.java

```diff
@@ -42,9 +42,12 @@
 @DocumentedFeature(groupName=HelpConstants.DOC_CAT_ANNOTATORS, groupSummary=HelpConstants.DOC_CAT_ANNOTATORS_SUMMARY, summary="Root mean square of the mapping quality of reads across all samples (MQ)")
 public final class RMSMappingQuality extends InfoFieldAnnotation implements StandardAnnotation, ReducibleAnnotation {
     private static final RMSMappingQuality instance = new RMSMappingQuality();
+    public static final int NUM_LIST_ENTRIES = 2;
+    public static final int SUM_OF_SQUARES_INDEX = 0;
+    public static final int TOTAL_DEPTH_INDEX = 1;
 
     @Override
-    public String getRawKeyName() { return GATKVCFConstants.RAW_RMS_MAPPING_QUALITY_KEY;}
+    public String getRawKeyName() { return GATKVCFConstants.RAW_MAPPING_QUALITY_WITH_DEPTH_KEY;}   //new key for the two-value MQ data to prevent version mismatch catastrophes
```

Comment:
Perhaps we could add a check and emit a warning to the user if there is a mismatch between the versions of the rmsMappingQualityKey in their files?

Reply:
I added a check in the finalize method in case GenotypeGVCFs gets run on old data.  GenomicsDB behavior is largely outside of our control here.  I don't have a great idea for CombineGVCFs though.  I thought about adding a check against ReducibleAnnotation.getDeprecatedKeyName(), but I feel like having getDeprecatedKeyName() return null for the other annotations and querying the annotationMap for null is an invitation for trouble.